config/dispatcher/dispatcher.ex

```diff
@@ -86,6 +86,10 @@ defmodule Dispatcher do
     forward(conn, path, "http://cache/bestuurseenheid-classificatie-codes/")
   end
 
+  match "/bestuurseenheid-contact/*path", %{layer: :resources, accept: %{json: true}} do
```

Comment:
very strange to make this singular when all the rest is plural though. Could you change it?

Reply:
To `bestuurseenheid-contacten`